### PR TITLE
Holy mother of parse

### PR DESCRIPTION
**Pull request by JesseStephenBrown**
I will add an updated LL(1) table to google docs to reflect changes to optionalactualparameterlist. I have not finished re-reviewing the whole LL(1) table, so we might find more crapola.

### CODE REVIEW

File: parser.cs [hunk 1]
```diff
@@ -777,8 +785,29 @@ public class Parser {
 
     private void optionalActualParameterList(){
```
Comment (brianpmaher) [inline parser.cs:786]: What led you to this fix?

File: parser.cs [hunk 2]
```diff
@@ -11,9 +11,14 @@ public class Parser {
     public Parser(List<Token> tokens){
         this.__tokens = tokens;
         this.__e = __tokens.GetEnumerator();
+        this.__e.MoveNext();
         this.__lookahead = __e.Current;
     }
 
+    public void Parse(){
```
Comment (brianpmaher) [inline parser.cs:18]: Nice call on this, totally forgot about it

**Comment by brianpmaher** [comment]
Looks good, +1. Also made a ticket for the LL(1) doc fix

File: parser.cs (hunk 1 above)
Comment (JesseStephenBrown) [inline parser.cs:786]: Tried to do a := b \* c and it didn't work. Then tried a := b \* 5 which also didn't work. So, either µPascal sucks hard, or we did something wrong.

**Comment by seanrogers2657** [comment]
Nice. +1 We should definitely get together and work through the table just to be sure we are doing it correctly.

**Pull request merged**